crates/pipeline-manager/src/license.rs

```diff
@@ -19,20 +19,38 @@ pub enum DisplaySchedule {
     Always,
 }
 
-#[derive(Serialize, Deserialize, ToSchema, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema)]
 pub struct LicenseInformation {
-    /// Timestamp at which the license expires
-    pub expires_at: DateTime<Utc>,
-    /// Whether the license is expired
-    pub is_expired: bool,
+    /// Timestamp when the server responded.
+    pub current: DateTime<Utc>,
+    /// Timestamp from which point the license is valid
+    pub valid_from: DateTime<Utc>,
```

Comment:
we just discussed this but it should essentially always be valid as soon as we hand it out so this one can be removed